.drone.yml

```diff
@@ -41,7 +41,7 @@ unit_tests: &unit_tests
 
 sonar_scanner: &sonar_scanner
   pull: if-not-exists
-  image: quay.io/ukhomeofficedigital/sonar-scanner-node:549b75f593f28da1c4a0a6f79877ec69d3b21037
+  image: quay.io/ukhomeofficedigital/sonar-scanner-nodejs:latest
```

Comment:
It’s usually good idea to avoid latest as it will keep updating the image for every deployment to the latest one. Ideally it’ll be good to take a specific sha